resolve/matrix/Sparse.cpp

```diff
@@ -149,6 +231,16 @@ namespace ReSolve { namespace matrix {
     return 0;
   }
 
+  /**
+   * @brief Set the pointers for matrix row, column, value data. Useful if interfacing with other codes - this function only assigns pointers, but it does not allocate nor copy anything. Note that the data ownership flags would be left at default (false).
+   *
+   * @param[in] row_data - pointer to row data (array of integers)
```

Comment:
Separate brief from the description:
```suggestion
  /**
   * @brief Set the pointers for matrix row, column, value data. 
   *
   * Useful if interfacing with other codes - this function only assigns pointers, 
   * but it does not allocate nor copy anything. Note that the data ownership flags
   * would be left at default (false).
   *
   * @param[in] row_data - pointer to row data (array of integers)
```